relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -922,17 +912,6 @@ mod tests {
 
     span_description_test!(db_prisma, "User find", "db.sql.prisma", "User find");
 
-    span_description_test!(
-        long_description_none,
-        // Do not attempt to parse very long descriptions.
-        {
-            let repeated = "+1".repeat(5000);
-            &("SELECT 1".to_string() + &repeated)
-        },
-        "db.query",
-        ""
-    );
```

Comment:
This constructed case still causes a stack overflow in the visitor, but we haven't seen those cases in the wild. Nevertheless, I will report this problem upstream.